.gitignore

```diff
@@ -1,3 +1,4 @@
+env
```

Comment:
Change to `/env/` to avoid capturing any non-directory env files that aren't at the top level.

Reply:
Done.